src/modules/home/HomePage.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import {
+  Grid,
+  Row,
+  Button,
+  form,
+  FormGroup,
+  FormControl,
+  InputGroup,
+  PageHeader,
+  Col
+} from 'react-bootstrap';
+import {I18n} from 'react-i18nify';
+import inject from 'react-rxjs/dist/RxInject';
+import FontAwesome from 'react-fontawesome';
+import './index.css';
+import Config from '../../config';
+import {hashHistory} from 'react-router';
+import appSession$ from './../../modules/app/appSession';
+import Logos from '../../components/logos/Logos';
+
+
+const reportURL = Config.magasin.urls.client.report.goToReport(appSession$);
+const magasinURL = Config.magasin.urls.client.magasin.goToMagasin(appSession$);
+const analysisURL = Config.magasin.urls.client.analysis.addAnalysis(appSession$);
+const aboutURL = '/about';
+const notFoundURL = '/notfound';
+
+const goTo = (url) => hashHistory.push(url);
+
+export const HomePage = () => {
+  const buttonAdd = (t, url)=> {
```

Comment:
three things:

* Convert to functional component aka ``` const AddButton = ({ t, url }) => { .... } ```
* Move outside HomePage
* remove curly body in HomePage